src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="starts__star"></div>
+      <div class="starts__star"></div>
+      <div class="starts__star"></div>
+      <div class="starts__star"></div>
+      <div class="starts__star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="starts__star is-active"></div>
+      <div class="starts__star"></div>
+      <div class="starts__star"></div>
+      <div class="starts__star"></div>
+      <div class="starts__star"></div>
+    </div>
+    <div class="stars stars--2">
+      <div class="starts__star is-active"></div>
+      <div class="starts__star is-active"></div>
+      <div class="starts__star"></div>
+      <div class="starts__star"></div>
+      <div class="starts__star"></div>
+    </div>
+    <div class="stars stars--3">
+      <div class="starts__star is-active"></div>
+      <div class="starts__star is-active"></div>
+      <div class="starts__star is-active"></div>
+      <div class="starts__star"></div>
+      <div class="starts__star"></div>
+    </div>
```

Comment:
1. Let's use your block modifier to change some stars inside the block to active. Remove all `is-active` classnames
2. Add some space between children to improve readability (checklist p1)
![image](https://github.com/mate-academy/layout_stars/assets/84296041/ce5344ce-23e8-4863-8685-d10f297dff32)

```suggestion
    <div class="stars stars--1">
      <div class="starts__star"></div>
      <div class="starts__star"></div>
      <div class="starts__star"></div>
      <div class="starts__star"></div>
      <div class="starts__star"></div>
    </div>
    
    <div class="stars stars--2">
      <div class="starts__star"></div>
      <div class="starts__star"></div>
      <div class="starts__star"></div>
      <div class="starts__star"></div>
      <div class="starts__star"></div>
    </div>
    
    <div class="stars stars--3">
      <div class="starts__star"></div>
      <div class="starts__star"></div>
      <div class="starts__star"></div>
      <div class="starts__star"></div>
      <div class="starts__star"></div>
    </div>
```